.tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml

```diff
@@ -226,7 +276,29 @@ spec:
                 value: "gpt-3.5-turbo-1106"
               - name: CP_OFFLINE_TOKEN
                 value: $(cat /var/run/insights-stage-upload-offline-token/token)
-            image: registry.redhat.io/openshift4/ose-cli:latest
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+              - name: OLS_IMAGE
+                value: $(params.serviceimage)
+              - name: ARTIFACT_DIR
+                value: "tests/test_results"
+              - name: SUITE_ID
+                value: "nosuite"
+              - name: TEST_TAGS
+                value: ""
+            image: registry.access.redhat.com/ubi9/ubi-minimal
             script: |
               echo "---------------------------------------------"
-              ls /var/run/azureopenai-entra-id
\ No newline at end of file
+              microdnf -y install git make python3.11 python3.11-devel python3.11-pip shadow-utils tar
+              curl -L -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.16/openshift-client-linux-amd64-rhel9.tar.gz \
+                  && tar -xvzf oc.tar.gz \
+                  && chmod +x kubectl oc \
+                  && mv oc kubectl /usr/local/bin/
+              git clone https://github.com/openshift/lightspeed-service.git
+              cd lightspeed-service
+              pip3.11 install --no-cache-dir --upgrade pip pdm mypy black ruff
+              pdm config python.use_venv false
+              make install-deps
+              make install-deps-test
+              pdm run pytest tests/e2e -s --durations=0 -o junit_suite_name="${SUITE_ID}" -m "${TEST_TAGS}" --junit-prefix="${SUITE_ID}" --junit-xml="${ARTIFACT_DIR}/junit_e2e_${SUITE_ID}.xml" \
```

Comment:
Why not `make test-e2e`?

Reply:
`make test-e2e` was causing a bunch of dependency issues:
python not installed
pytest not installed
ReportPortalClient not installed
Basically everything that was dependent on pdm was not working. So, due to the short timeframe we have for this, I decided to add the `pdm run pytest ...`